tests/Microsoft.Identity.Web.Test.Common/TestConstants.cs

```diff
@@ -55,6 +55,8 @@ public static class TestConstants
         public const string GraphBaseUrlBeta = "https://graph.microsoft.com/beta";
         public const string GraphBaseUrl = "https://graph.microsoft.com/v1.0";
 
+        public const string IdToken = "ewogICJ0eXAiOiAiSldUIiwKICAiYWxnIjogIlJTMjU2IiwKICAia2lkIjogImsiCn0=.ewogICJuYW1lIjogIlVzZXIiCn0=.";
```

Comment:
?

Reply:
Needed to have a well-constructed token for the test to run.